### DISCUSSION TITLE
Can a IP PTZ surveillance camera be controlled through Stellarium?

### DISCUSSION THREAD

**Discussion by Robertomcat** [Stellarium/stellarium#3131] [Ideas]
Hello, good afternoon.

I am a guy with a physical disability due to a traffic accident, and as I have lost most of my mobility I cannot use a telescope, let alone take it with me to the mountains to make visualizations.

I am a metrology amateur (basic) and I have a camera installed next to the weather station that sends quality data to AVAMET (Valencian Association of Meteorology) and I am going to incorporate another PTZ camera to be able to visualize the components of the universe (only the brightest ones).

That's why I wanted to ask if there is a way to manage this kind of cameras through stellarium, it would be very enriching for me and for many other people to be able to make use of this alternative. 

This is the camera I am going to incorporate.
The RS-485 protocol is used to move the camera's XYZ axis and zoom, and the baud rate can also be changed on the camera.

![1_0_01_07_14224_8771814932_thumb](https://user-images.githubusercontent.com/17690060/227958130-8f5eb89b-a75a-486e-b516-04fcd5837f83.png)
![image](https://user-images.githubusercontent.com/17690060/227971582-29fa6f34-916f-45ee-a89f-36f7f50e4d39.png)
![image](https://user-images.githubusercontent.com/17690060/227971771-8369a04a-4eb0-4758-bb8f-73494c163bf3.png)

Thank you very much.

**Comment by gzotti** [comment]
Hello!

There is no direct support for such cameras in Stellarium. Maybe ask in amateur astronomy forums for ideas/advice?

**Reply by Robertomcat** [reply]
Do you mean change the post category to "ideas", or look at alternative forums to this one?

**Reply by gzotti** [reply]
The latter. Sorry, we are about 3 developers, and will likely not have enough free time to support such extras not used by ourselves.

**Reply by Robertomcat** [reply]
Yes, I understand perfectly well. If anyone has any ideas related to this issue I have raised, it would be of great help. Thanks for the work you do in stellarium.
